connect/cli/plugins/commerce/utils.py

```diff
@@ -0,0 +1,340 @@
+        ws.column_dimensions[column].width = 22
+    ws.column_dimensions['H'].width = 40
+    _fill_headers(
+        ws,
+        (
+            'ID',
+            'Function ID',
+            'Function Name',
+            'Description',
+            'Overview',
+            'Input columns',
+            'Output Columns',
+            'Position',
+            'Settings',
+        ),
+    )
+    alignment = Alignment(
+        horizontal='left',
+        vertical='top',
+    )
+    task = progress.add_task('Filling transformations', total=len(transformations))
+    for n, transformation in enumerate(transformations, 2):
+        ws.cell(n, 1, value=transformation['id']).alignment = alignment
+        ws.cell(n, 2, value=transformation['function']['id']).alignment = alignment
+        ws.cell(n, 3, value=transformation['function']['name']).alignment = alignment
+        ws.cell(n, 4, value=transformation.get('description', '')).alignment = alignment
+        ws.cell(n, 5, value=transformation['overview']).alignment = alignment
+        col_alignment = Alignment(
+            horizontal='left',
+            vertical='top',
+            wrap_text=True,
+        )
+        ws.cell(
+            n,
+            6,
+            value='\n'.join([c['id'] for c in transformation['columns']['input']]),
+        ).alignment = col_alignment
+        ws.cell(
+            n,
+            7,
+            value='\n'.join([c['id'] for c in transformation['columns']['output']]),
+        ).alignment = col_alignment
+        ws.cell(n, 8, value=transformation['position']).alignment = alignment
+        ws.cell(
+            n,
+            9,
+            value=json.dumps(transformation['settings'], indent=4, sort_keys=True),
+        ).alignment = col_alignment
+        ws.row_dimensions[n].height = 120
+        progress.update(task, advance=1)
+
+
+def _download_file(client, folder_type, folder_name, file_id, file_destination):
+    try:
+        response = (
+            client.ns(
+                'media',
+            )
+            .ns(
+                'folders',
+            )
+            .collection(
+                folder_type,
+            )[folder_name]
+            .collection(
+                'files',
+            )[file_id]
+            .get()
+        )
+    except ClientError:
+        raise ClickException(f'Error obtaining file {file_id} -> {file_destination}')
+    with open(file_destination, 'wb') as f:
+        f.write(response)
+    console.secho(
+        f'File {file_destination} downloaded properly.',
+        fg='green',
+    )
+
+
+def fill_and_download_attachments(ws, attachments, client, base_path, progress):
+    attachments = list(attachments)
+    ws.column_dimensions['A'].width = 30
+    ws.column_dimensions['B'].width = 30
+    _fill_headers(
+        ws,
+        ('ID', 'Name'),
+    )
+    task = progress.add_task('Filling and downloading attachments', total=len(attachments))
```

Comment:
Before adding a progress, if the len of attachments is 0 you should return from this funciton